### DISCUSSION TITLE
Is the project stopped?

### DISCUSSION THREAD

**Discussion by brnosouza** [python-gino/gino#799] [Q&A]
I see that the latest commit is from last year because we were waiting for SQLAlchemy 1.4, but it was released, is there anything holding the next release?

**Comment by fantix** [comment]
Do you mean in the [v2.0.x branch](https://github.com/python-gino/gino/commits/v2.0.x)? Yeah I haven't got enough time to work on SQLAlchemy 1.4 support recently. But bugfixes are still expected on the current 1.0/1.1 stream.

**Reply by brnosouza** [reply]
Thanks

**Reply by Pentusha** [reply]
@fantix any chance to see already published features of 1.1.0 on PyPI without pre-release chevron?
Also, It would be nice if you will publish already merged features (e.g. python-gino/gino#786)

**Comment by aprilmay** [comment]
There hasn't been a single commit since Feb. 2022, and compatibility issues start piling up dangerously.
@fantix, can you please post an update where you stand about Gino's development? Thanks!

**Comment by akalex** [comment]
Hey @fantix, I just to reiterate an initial question to better understand what's the status of the Gino project. Is there a plan to add support for SQLAlchemy 1.4+? Many thanks for considering my request.
